kmip/demos/utils.py

```diff
@@ -255,9 +273,61 @@ def build_cryptographic_algorithm(logger, object_type):
         sys.exit()
 
 
-def build_key_value(logger, object_type):
-    if (object_type == ObjectType.SYMMETRIC_KEY
-       or object_type == ObjectType.SECRET_DATA):
+def build_secret_value(logger, object_type):
```

Comment:
Can you add a note on how you generated this like for the public and private key?